circle.yml

```diff
@@ -6,7 +6,9 @@ machine:
 
 dependencies:
   override:
+    - brew install swiftlint
     - bundle install
+    - rake danger
```

Comment:
Ah, yeah, this is a bug in Danger - https://github.com/danger/danger/blob/master/lib/danger/ci_source/circle.rb#L25

Needs to have that top check for the PR_ID removed